include/dg/llvm/CallGraph/CallGraph.h

```diff
@@ -15,6 +15,8 @@
 #include "dg/PointerAnalysis/PSNode.h"
 #include "dg/llvm/PointerAnalysis/PointerAnalysis.h"
 
+#include "llvm/llvm-utils.h"
+
```

Comment:
This is a problem, because "llvm/llvm-utils.h" is a private header. It is not installed.

Reply:
You're right, I'm going to drop f280b7c229c04cee5e5bf6f66b7b3a6783743277 for now and I can't think of any other solution to this than to create a new library and to move the appropriate code out of this header file.

Reply:
Yes, there is probably no other way.